internal/controller/telemetry/otel_conf_gen.go

```diff
@@ -288,12 +415,52 @@ func (cfgInput *OtelColConfigInput) generateConnectors() map[string]any {
 
 }
 
+func generateCountConnectors() map[string]any {
```

Comment:
we also need:
- namespace
- node name
- container name
- app label -> we have a "smart" solution for this in logging operator with syslog-ng, but I'm not sure we can replicate that with otel

Reply:
Working on a change to enable count connector to count by resource attributes, to be able to do this without dirty workarounds.